src/openrct2/object/ObjectRepository.cpp

```diff
@@ -98,13 +98,12 @@ class ObjectRepository final : public IObjectRepository
     std::vector<ObjectRepositoryItem>   _items;
     QueryDirectoryResult                _queryDirectoryResult = { 0 };
     ObjectEntryMap                      _itemMap;
-    uint16                              _languageId = 0;
-    sint32                                 _numConflicts;
+    uint16                              _languageId   = 0;
+    sint32                              _numConflicts = 0;
 
 public:
-    ObjectRepository(IPlatformEnvironment * env)
+    ObjectRepository(IPlatformEnvironment * env) : _env(env)
     {
-        _env = env;
     }
```

Comment:
I have to disagree. I understand your point about syntax, but having this as part of initialization list provides better hints to compiler. I remember you raising complaints about code being generated where it shouldn't have been, that's one of the ways to ensure that.